src/ShortNumbers.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Scanner;
+
+public class ShortNumbers {
+    public static void main(String[] args) {
+
+        // Find the arithmetic mean
+        Scanner myObj = new Scanner(System.in);
+        System.out.println("Enter your digits,please! ");
+        short a = myObj.nextShort();
+        short b = myObj.nextShort();
+        short c = myObj.nextShort();
+        int result = (a + b + c) / 3;
+
+        int arithmeticMean = 0;
+
+        if (a < 0) {
+            arithmeticMean = arithmeticMean + (-a);
+        } else {
+            arithmeticMean = arithmeticMean + a;
+        }
+
+        if (b < 0) {
+            arithmeticMean = arithmeticMean + (-b);
+        } else {
+            arithmeticMean = arithmeticMean + b;
+        }
+
+        if (c < 0) {
+            arithmeticMean = arithmeticMean + (-c);
+        } else {
+            arithmeticMean = arithmeticMean + c;
+        }
```

Comment:
You gave wrong names to your calculation results. Arithmetic mean is itself `(a + b + c) / 3`, but you gave that name to the calculation results of the sum of all the absolute values of 3 inputs.
Change variable names, so that it won't be confusing and reflect real content of the value they hold.